scripts/lint_naming_conventions.py

```diff
@@ -89,6 +93,22 @@ def parse_file_name(in_dir, in_file):
     return 0
 
 
+# Ensure all jobs have a retention policy
```

Comment:
If you are going to add the functionality to this file, I think you should rename it and modify any references to it being specific to naming conventions because it and its use in lint.sh is a bit misleading.

Reply:
Done